src/ui/components/Comments/TranscriptComments/CommentEditor/ExistingCommentEditor.tsx

```diff
@@ -43,11 +43,11 @@ function ExistingCommentEditor({
     commentId: comment.id,
   });
 
-  const handleSubmit = (inputValue: string) => {
+  const handleSubmit = async (inputValue: string) => {
     if (type === "comment") {
-      updateComment(comment.id, inputValue, (comment as Comment).position);
+      await updateComment(comment.id, inputValue, (comment as Comment).position);
     } else if (type === "reply") {
-      updateCommentReply(comment.id, inputValue);
+      await updateCommentReply(comment.id, inputValue);
```

Comment:
Do I correctly understand that we just synchronously wait? Thus bypassing React rerenders in the meantime, thus avoiding flicker?

Reply:
I'm actually a bit confused why we have the call to `clearPendingComment` below.  I added this change to be consistent with `NewCommentEditor`, but if we can remove the call to `clearPendingComment` it is probably not necessary.